### PR TITLE
Page selector for data table

### PR DESCRIPTION
**Pull request by lukicenturi**
Closes #(issue_number)

## Checklist

image

### CODE REVIEW

**Comment by yabirgb** [comment]
In my opinion with this change the experience is much better than before 
image

**Review by yabirgb — COMMENTED** [review]
Linting failed. For the case where we have 0 events maybe you can add an exception to improve this? 
image

**Comment by lukicenturi** [comment]
> Linting failed. For the case where we have 0 events maybe you can add an exception to improve this? image

image

**Review by yabirgb — APPROVED** [review]
Looks good and CI is okeym, merging. Thanks!

File: frontend/app/src/components/helper/DataTable.vue [hunk 1]
```diff
@@ -148,5 +214,28 @@ const scrollToTop = () => {
   }
 }
 
+.items-page-select {
+  display: flex;
+
+  :deep() {
```
Comment (kelsos) [inline frontend/app/src/components/helper/DataTable.vue:220]: Btw I think we are using the new `:deep()` selector wrong.

https://vuejs.org/api/sfc-css-features.html#deep-selectors

I am getting some warnings in WebStorm but also the example suggests `:deep(.v-input) {...}`

**Pull request merged**